### ISSUE TITLE
A bug when there is no point satisfy disp < maxdisp

### ISSUE THREAD

**Issue by youmi-zym** [JiaRenChang/PSMNet#90]
Hi:
I find there are some images' disparity of SceneFlow are bigger than maxdisp,
```
mask = disp_true < args.maxdisp
mask.detach_()
```
mask is all zero here, there will be a error when [calculating loss](https://github.com/JiaRenChang/PSMNet/blob/master/main.py#L94),  and it can't  be backward. 

An example:
![image](https://user-images.githubusercontent.com/17737537/47950363-2bea7000-df8c-11e8-81f9-9f67226016ec.png)

The code will run well with batch-size bigger than 1, but batch-size=1 will lead this bug

my advice is before model forward, check wether mask is all 0:

```
mask = (disp_true < args.max_disp)
mask = mask.detach().type_as(disp_true)
if mask.sum() < 1.0:
    continue
```

**Comment by JiaRenChang** [comment]
Hi, @youmi-zym 
Thank you. I will modified it.
